docs/docs/explore/project-chat.md

```diff
@@ -0,0 +1,169 @@
+   - Time series charts showing trends over time
+   - Bar charts comparing dimensions or categories
+   - Tables with formatted data
+   - Big number displays for key metrics
+3. **Detailed Results** - Tables or lists with specific numbers and breakdowns
+4. **Dashboard Links** - Direct links to your existing [Explore dashboards](/explore/dashboard-101) with filters and time ranges pre-applied based on your question
+5. **Suggested Next Steps** - Follow-up questions or areas to investigate further
+
+### Linking Back to Dashboards
+
+One of the most powerful features of AI Chat is how it seamlessly connects back to your existing Rill dashboards. When the AI answers your question, it automatically generates links that:
+
+- **Pre-apply relevant filters** - The dashboard opens with filters matching your question context
+- **Set appropriate time ranges** - Time periods from your question are automatically selected
+- **Select relevant metrics** - The measures and dimensions discussed in the chat are highlighted
+- **Enable comparison periods** - When you ask about changes, comparison views are activated
+
+This means you can **start with a quick AI answer**, then **click through to the full dashboard** for deeper, interactive exploration - all without manually setting up filters or searching for the right view.
+
+### Visual Components
+
+Unlike text-only AI assistants, AI Chat in Rill Cloud can render actual dashboard components directly in the chat interface. These visualizations are built using the same [canvas dashboard](/build/dashboards/canvas-widgets) technology used throughout Rill, ensuring:
+
+- **Consistency** - Visualizations match the style and behavior of your regular dashboards
+- **Interactivity** - Click, hover, and interact with charts right in the chat
+- **Accuracy** - Charts are generated from the same data sources as your dashboards
+- **Clarity** - Complex data patterns become immediately visible
+
+:::tip From Chat to Dashboard
+Think of AI Chat as your starting point for data exploration. Use it to quickly answer questions and spot trends, then leverage the dashboard links to dive deeper with full filtering, comparison, and export capabilities.
+:::
+
+## Improving AI Chat with Instructions
+
+To get the most accurate and contextual responses from AI Chat, you can add custom `ai_instructions` to your project files. These instructions provide the AI with additional context about your data, business logic, and preferred response formats.
+
+### Why Add AI Instructions?
+
+AI instructions help the AI:
+- Understand your specific business context and terminology
+- Format responses in ways that match your team's preferences
+- Generate properly formatted Explore dashboard links
+- Focus on the metrics and dimensions most relevant to your use case
+- Provide more actionable insights tailored to your workflows
```

Comment:
I'm not sure what this one means. I think it's kind of covered in the above bullets?